exposed-core/src/main/kotlin/org/jetbrains/exposed/sql/Queries.kt

```diff
@@ -269,7 +269,10 @@ private fun FieldSet.selectBatched(
         }
 
         private fun toLong(autoIncVal: Any): Long = when (autoIncVal) {
-            is EntityID<*> -> autoIncVal.value as Long
+            is EntityID<*> -> when (val value = autoIncVal.value) {
+                is Int ->  value.toLong()
```

Comment:
not a maintainer, but that double space triggers my OCD :cry: 

```suggestion
                is Int -> value.toLong()
```